radix-engine-tests/assets/cost_transfer_to_virtual_account.csv

```diff
@@ -20,9 +20,8 @@ Execution Cost Breakdown                                                   ,
 - OpenSubstate::GlobalNonFungibleResourceManager                           ,                    42414,      1.1%
 - OpenSubstate::GlobalPackage                                              ,                  2038410,     51.8%
 - OpenSubstate::GlobalVirtualSecp256k1Account                              ,                   486107,     12.4%
-- OpenSubstate::InternalAccount                                            ,                      786,      0.0%
 - OpenSubstate::InternalFungibleVault                                      ,                    94951,      2.4%
-- OpenSubstate::InternalGenericComponent                                   ,                    50713,      1.3%
```

Comment:
I wonder why this has increased in size by the above? o.O

Reply:
During account virtualization,  opening the substates is grouped into `InternalGenericComponent`.